src/harness/fake_sas.py

```diff
@@ -263,6 +266,9 @@ def TriggerPpaCreation(self, request, ssl_cert=None, ssl_key=None):
   def TriggerDailyActivitiesImmediately(self):
     pass
 
+  def TriggerEnableNTIAExclusionZones(self):
```

Comment:
Please use `Ntia` (only the first letter capitalized) to match with the style of the nearby code.

Reply:
Removed EXZ.2 from this PR. We will submit another PR once EXZ.1 get approved. We will change it to Ntia.